post.md

```diff
@@ -0,0 +1,226 @@
+# Robot Framework - using Request library for control RPI GPIO's
+
+Requests library is one of the most popular libraries implemented for Robot
+Framework. It is very important for testing scattered applications, so this is
```

Comment:
@konopskyyyy scattered ? Maybe you meant distributed ?

http://www.dictionary.com/browse/scattered